### PR TITLE
Change connect:auth to connect:sf:auth

### PR DESCRIPTION
**Pull request by gulopine**
This better segments the Salesforce behavior into a namespace that can be reused for more features later on.

### CODE REVIEW

File: lib/commands/connect/sf-auth.js [hunk 1]
```diff
@@ -20,9 +20,9 @@ function callbackServer() {
 
 module.exports = {
   topic: 'connect',
-  command: 'auth',
-  description: 'get Salesforce authorization URL',
-  help: 'return an URL to authorize a connection to a Salesforce Org',
+  command: 'sf:auth',
+  description: 'Authorize access to Salesforce for your connection',
+  help: 'Provides a URL to authorize a connection to a Salesforce Org',
```
Comment (sibson) [inline lib/commands/connect/sf-auth.js]: is the help correct here?  seems odd.

Comment (gulopine) [inline lib/commands/connect/sf-auth.js]: Well, technically it does both (shows a URL and opens a browser to said URL), and I was just trying to find a way to make the help different from the description, so it wasn't just a duplication. But yeah, displaying the URL isn't really the purpose, so maybe that doesn't make sense.

**Comment by sibson** [comment]
+1

**Comment by sibson** [comment]
:thumbsup: :shipit:

**Pull request merged**